modules/oe_editorial_content_lock/oe_editorial_content_lock.install

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Editorial Content Lock module.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Implements hook_install().
+ */
+function oe_editorial_content_lock_install() {
+  // Set default settings for Content Lock.
+  $locked_contents = [
+    'file' => [
+      '*' => '*',
```

Comment:
This kind of contents is not tested on the behat test.

Reply:
I updated your other comments but I'm not sure how to tackle this. I didn't find any means of editing a file in core D8 so I'm not sure if we should even include it. What do you think?